build.gradle

```
@@ -13,7 +13,7 @@ repositories {
 
 dependencies {
 
-    runtime files ('target/sync-client-2.1.1.jar','target/sync-client-2.1.1-javadoc.jar','target/sync-client-2.1.1-sources.jar')
+    runtime files ('dependencies/sync-client-2.1.1.jar','dependencies/sync-client-2.1.1-javadoc.jar','dependencies/sync-client-2.1.1-sources.jar')
```

It doesn't compile with `runtime`
```suggestion
    implementation files ('dependencies/sync-client-2.1.1.jar','dependencies/sync-client-2.1.1-javadoc.jar','dependencies/sync-client-2.1.1-sources.jar')
```